api/managementpb/dbaas/components.proto

```diff
@@ -102,6 +102,43 @@ message ChangePXCComponentsRequest {
 
 message ChangePXCComponentsResponse {}
 
+// OperatorUpdateStatus represents status of an update for the operator.
+enum OperatorUpdateStatus {
+  OPERATOR_UPDATE_STATUS_INVALID = 0;
+  // An update of the operator is available for the installed version of PMM server.
+  UPDATE_AVAILABLE = 1;
+  // An update of the operator is not available for the installed version of PMM server.
+  UPDATE_NOT_AVAILABLE = 2;
+  // An update of the operator is available but for more up to date version of
+  // PMM server than is currently installed.
+  UPDATE_AVAILABLE_BUT_NOT_COMPATIBLE = 3;
+}
+
+message CheckForOperatorUpdateRequest {
+  // Kubernetes cluster name.
+  string kubernetes_cluster_name = 1 [
+    (validator.field) = {
+      string_not_empty: true
+    }
+  ];
```

Comment:
When we are going to check for operator update? How often?

Reply:
If it's the same as list kubernetes cluster, then we can just extend ListKubernetesCluster.

Reply:
Or just send make one endpoint for all clusters to not send N requests to version service.

Reply:
Calling version service only once, only one API endpoint to get information about all updates.